google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java

```diff
@@ -596,25 +593,20 @@ public QueryResponse runQuery(String query) throws InterruptedException {
   /**
    * Example of running a query with query parameters.
    */
-  // [TARGET query(QueryRequest)]
+  // [TARGET query(QueryJobConfiguration, QueryOption...)]
   // [VARIABLE "SELECT distinct(corpus) FROM `bigquery-public-data.samples.shakespeare` where word_count > @wordCount"]
   public QueryResponse runQueryWithParameters(String query) throws InterruptedException {
     // [START runQueryWithParameters]
-    QueryRequest request = QueryRequest.newBuilder(query)
+    QueryJobConfiguration queryConfig = QueryJobConfiguration.newBuilder(query)
         .setUseLegacySql(false) // standard SQL is required to use query parameters
```

Comment:
No longer required to explicitly set legacy to false now, correct?

Reply:
Yes. I removed the setter but left and edited the corresponding comment (to emphasize that parameters work only with Std SQL).